### DISCUSSION TITLE
can tags be implemented?

### DISCUSSION THREAD

**Discussion by eyck-dev** [gilcu3/hullcaster#4] [Ideas]
I find it a bit clumsy to go through all my feeds on the list, i wish there was a tags feature to filter them

**Comment by gilcu3** [comment]
Certainly that would be a nice addition. For the moment it will be possible to show a list of unplayed episodes, see #3. Tags should be implemented in the future, but will not be a priority for now, unless someone else implements them. 

Just for curiosity, how many podcasts do you have? (I am assuming that the need for tags appears after a certain number of them.)

**Reply by eyck-dev** [reply]
I have 18-20 podcasts, I arrange them as "games", "movies", etc. Sorry for the late response.
